SwiftWisdom/Core/UIKit/UIView+Geometry.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  UIView+Geometry.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 1/19/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    public var ip_width: CGFloat {
+        return CGRectGetWidth(frame)
```

Comment:
@brightredchilli I'm really torn on this one, there's a couple considerations. My main one is that whenever I'm doing any programmatic layout calculations, I use these methods so much that even with the other accessors, I always end up adding them.

I think the main motivation for the `frameX`, `boundsMidX` naming stuff is because setters are involved, so we need to be a bit more articulate.

With the properties here, they're access only so I think they're safe enough to include.

Thoughts on this?